cmd/kata-manager/kata-manager.sh

```diff
@@ -327,8 +328,13 @@ exec_document()
 
 	info "$msg"
 
-	# run the installation
-	bash "${install_script}"
+	# run the installation, if no argument for release is used, the
+	# default branch will be master from the install script
+	if [ -z "$release" ]; then
+		bash "${install_script}"
+	else
+		BRANCH="$release" bash "${install_script}"
```

Comment:
Hi @cmaf - I'm not quite sure how this would work. You could possibly pass `$release` as an extra param to `get_git_repo()` so that it would `git clone` and then `git checkout` the specified branch?

Reply:
@jodh-intel the [install scripts](https://github.com/kata-containers/documentation/blob/master/install/ubuntu-installation-guide.md) specify master in all branches. Also `get_git_repo()` is called only for tests and documentation. It doesn't appear that passing it to `get_git_repo()` would help. Unless I'm missing something - could you clarify?

Reply:
Aha! Sorry @cmaf - I hadn't actually checked the install docs! This is fine then ;)

Reply:
Might be worth adding a comment to make it manifest that this code relies on the installation docs using that variable.